docs/standard/commandline/get-started-tutorial.md

```diff
@@ -0,0 +1,434 @@
+---
+title: "Tutorial: Get started with System.CommandLine"
+description: Learn how to use the System.CommandLine library for command-line apps.
+ms.date: 02/14/2022
+ms.topic: tutorial
+no-loc: [System.CommandLine]
+helpviewer_keywords:
+  - "command line interface"
+  - "command line"
+  - "System.CommandLine"
+---
+# Tutorial: Get started with System.CommandLine
+
+[!INCLUDE [scl-preview](../../../includes/scl-preview.md)]
+
+This tutorial shows how to create a .NET command-line app that uses the [`System.CommandLine` library](index.md). You'll begin by creating a simple root command that has one option. Then you'll add to that base, creating a more complex app that contains multiple subcommands and different options for each command.
+
+In this tutorial, you learn how to:
+
+> [!div class="checklist"]
+>
+> * Create commands, options, and arguments.
+> * Specify default values for options.
+> * Assign options and arguments to commands.
+> * Assign an option recursively to all subcommands under a command.
+> * Work with multiple levels of nested subcommands.
+> * Create aliases for commands and options.
+> * Work with `string`, `string[]`, `int`, `bool`, `FileInfo` and enum option types.
+> * Bind option values to command handler code.
+> * Use custom code for parsing and validating options.
+
+## Prerequisites
+
+* A code editor, such as [Visual Studio Code](https://code.visualstudio.com/) with the [C# extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp).
```

Comment:
Should we include VS here? If the user has VS, they do not need VSCode or the extension or the .NET SDK